setup/system.sh

```diff
@@ -110,10 +110,6 @@ hide_output add-apt-repository -y universe
 # Install the duplicity PPA.
 hide_output add-apt-repository -y ppa:duplicity-team/duplicity-release-git
 
-# Stock PHP is now 8.1, but we're transitioning through 8.0 because
-# of Nextcloud.
-hide_output add-apt-repository --y ppa:ondrej/php
```

Comment:
Didn't we need this repo for php8.0?  If https://github.com/mail-in-a-box/mailinabox/pull/2309/files#diff-82e1be7bfd6ea28b0bc7b8ee6ae4a78dcaea281159e0853b3039b97fc231cbceR232 is being used, this probably is needed too.